test/test_browser.py

```diff
@@ -3425,14 +3435,19 @@ def test_sdl2_mixer_wav(self, flags):
   @requires_sound_hardware
   def test_sdl2_mixer_music(self, formats, flags, music_name):
     shutil.copyfile(test_file('sounds', music_name), music_name)
-    self.btest_exit('test_sdl2_mixer_music.c', args=[
+    args = [
       '--preload-file', music_name,
       '-DSOUND_PATH="%s"' % music_name,
       '-DFLAGS=' + flags,
       '-sUSE_SDL=2',
       '-sUSE_SDL_MIXER=2',
       '-sSDL2_MIXER_FORMATS=' + ','.join(formats),
-    ])
+    ]
+    # libmodplug is written in C++ so we need to link in C++
+    # libraries when using it.
+    if 'mod' in formats:
+      args += ['-lc++', '-lc++abi']
```

Comment:
Would it be simpler maybe to make this a cpp test rather than c? I don't feel strongly though. I guess this is the workaround a C programmer would need, anyhow.

Reply:
I think we its nice to make sure that libsdl-mixer can be used from C programs.

As you know I really like not have C++ in the mix when debugging (unless the specific test needs C++).   Especially for debugging the linker its much easier not to have to see (the very long) C++ symbols in logging/tracing.